docs/data/toolpad/concepts/queries.md

```diff
@@ -1,53 +1,47 @@
-# Queries
+# Connecting to data
 
-<p class="description">Be it a database table or an external API, Toolpad offers mechanisms to bring server-side data to the page.</p>
-
-You can create two kinds of **queries** in Toolpad to bring data to your page.
-
-1. HTTP request
-2. Custom functions
+<p class="description">Be it a database table or an external API, Toolpad offers mechanisms to bring server-side data to the page as well as make mutations to external data sources. </p>
 
 :::info
-Toolpad uses `react-query` internally to run queries. Look into [its documentation](https://tanstack.com/query/latest/docs/react/guides/queries) for more details on query objects.
+Toolpad uses `react-query` internally to run queries and actions. Look into [its documentation](https://tanstack.com/query/latest/docs/react/guides/queries) for more details on query objects.
 :::
 
-## Mode
+## Queries
 
-You can set the **mode** of the query to either be automatically refetched on page load, or only be called on manual action.
+Queries allow you to bring backend data to your Toolpad page. They are called automatically on page load, so that data is available as state on the page as soon as the user interacts with it. You can modify the following settings for queries:
```

Comment:
```suggestion
Queries allow you to bring backend data to your Toolpad page. They are called automatically on page load, so that data is available as state on the page as soon as the user interacts with it. Toolpad will cache and regularly refresh the data. This means that your backend function will be called more than once. Queries are not suitable for backend functions that modify data. You can modify the following settings for queries:
```